app/exchanges/run_csv/pacbio_sample_sheet.rb

```diff
@@ -91,7 +91,7 @@ def sample_data(well, sample)
         bio_sample_name(sample),
         well.plate_well_position,
         sample.adapter, # left adapter
-        sample.adapter  # right adapter
+        sample.library&.tag&.oligo_reverse.presence || sample.adapter # right adapter
```

Comment:
This will return the reverse oligo instead of the group_id. Just wanted to check this is intended as the current adapter returns the group_id.

I think this logic should be moved to a method like above e.g. sample.right_adapter. 

Do we need to do .library? Sample in this case should be an aliquot which might already have the tag. If we do just library this may not work correctly for pools.